beets/autotag/hooks.py

```diff
@@ -156,8 +156,9 @@ def __init__(self, title, track_id, artist=None, artist_id=None,
                  length=None, index=None, medium=None, medium_index=None,
                  medium_total=None, artist_sort=None, disctitle=None,
                  artist_credit=None, data_source=None, data_url=None,
-                 media=None, lyricist=None, composer=None, composer_sort=None,
-                 arranger=None, track_alt=None):
+                 media=None, lyricist=None, lyricist_sort=None,
+                 composer=None, composer_sort=None, arranger=None,
+                 arranger_sort=None, track_alt=None):
```

Comment:
I think the problem's here: you're missing a line like this.

```py
self.lyricist_sort = lyricist_sort
```

Eventually, I've been meaning to refactor these classes here so all this duplication isn't necessary…